tests/com/twitter/intellij/pants/integration/OSSPantsCompileActionsTest.java

```diff
@@ -77,6 +86,36 @@ public void testCompileAllTargetsInModuleAction() throws Throwable {
     assertFalse(compileAllTargetsInModuleAction.doCleanAll());
   }
 
+  public void testCompileTargetsInSelectedEditor() throws Throwable {
+    doImport("examples/tests/scala/org/pantsbuild/example");
```

Comment:
Could you add another test that covers what happens if there are multiple editor panes open?